COLLABORATOR_GUIDE.md

```diff
@@ -294,22 +288,121 @@ After generating one, make sure to keep it safe because GitHub will not show the
 
 ### Doing a release
 
-To do a release, check out the master branch
+It is also recommended you have a clean clone of Video.js for each release line you want to relase.
```

Comment:
~~relase~~ -> release 
Maybe it would be good to specify why? I would think its because of node_modules and the possibility of HEAD not actually being the same as `origin/master`